TESTING.md

```diff
@@ -0,0 +1,4 @@
+Just look at how the github action CI is launched and reproduce it.
```

Comment:
OPT: add an example to test on test suite for example, and describe pre-requisites.